networks/semantic_segmentation_adas_0001/Makefile

```diff
@@ -0,0 +1,132 @@
+
+.PHONY: get_ir
+get_ir:	model_zoo
+	@echo $(YELLOW)'\n'${NETWORK_NAME}": Downloading IRs..."$(NOCOLOR);
+	@if [ -e ${ZOO_RELATIVE_DIR}/${MODEL_DOWNLOADER_DIR}/${SEG_MODEL_ZOO_DIR}/${MODEL_FILE_NAME_BASE}.xml ] || [ -e ${ZOO_RELATIVE_DIR}/${MODEL_DOWNLOADER_DIR}/${SEG_MODEL_ZOO_DIR}/${MODEL_FILE_NAME_BASE}.bin ]; \
+	then \
+		echo " - Model files already exists in open model directory."; \
+	else \
+		echo " - Model files do not exist in open model directory. ";\
+		echo " - Using Model downloader to download the model..."; \
+		(cd ${ZOO_RELATIVE_DIR}/${MODEL_DOWNLOADER_DIR}; python3 ${MODEL_DOWNLOADER_FILENAME} --name ${MODEL_FILE_NAME_BASE};); \
+	fi
+	
+	@if [ -e ${MODEL_FILE_NAME_BASE}.xml ] && [ -e ${MODEL_FILE_NAME_BASE}.bin ]; \
+	then \
+		echo " - IR files already exist in project directory. No need to copy IRs."; \
+	else \
+		echo " - IR files do not exist in project directory.";\
+		echo " - Copying IR files to current folder..."; \
+		cp ${ZOO_RELATIVE_DIR}/${MODEL_DOWNLOADER_DIR}/${SEG_MODEL_ZOO_DIR}/${MODEL_FILE_NAME_BASE}.xml .; \
+		cp ${ZOO_RELATIVE_DIR}/${MODEL_DOWNLOADER_DIR}/${SEG_MODEL_ZOO_DIR}/${MODEL_FILE_NAME_BASE}.bin .; \
+	fi
+
+
+.PHONY:	compile_cpp
+compile_cpp:  
+	@echo $(YELLOW)'\n'${APP_NAME}": Building the sample..."$(NOCOLOR)
+	@echo "Checking OpenVINO environment..."
+	@if [ -z "$(INTEL_OPENVINO_DIR)" ] ; \
+	then \
+		echo "Please initiate the Intel OpenVINO environment by going to the installation directory for openvino and running the setupvars.sh file in the bin folder." ; \
+		exit 1 ; \
+	else \
+		echo "Intel OpenVINO environment is already set!" ; \
+	fi
+	(mkdir -p build; cd build; cmake ..; make;)
+
+
+.PHONY: run
+run: run_cpp
+	
+
+.PHONY: run_cpp
+run_cpp: deps data compile_cpp
+	@echo $(YELLOW)'\n'${APP_NAME}": Running the sample..."$(NOCOLOR)
+	(cd build; ./${APP_NAME} ../people-brasil-guys-avpaulista-109919.jpg; cd ..;)
+
+
+.PHONY: install-reqs
+install-reqs: 
+	@echo $(YELLOW)"\n"$(APP_NAME)": Checking installation requirements..."$(NOCOLOR)
+	@echo "No requirements needed."
+
+
+.PHONY: uninstall-reqs
+uninstall-reqs: 
+	@echo $(YELLOW)'\n'${APP_NAME}": Uninstalling requirements..."$(NOCOLOR)
+	@echo "Nothing to uninstall."
+
+.PHONY: help
+help:
+	@echo "\nPossible make targets: ";
+	@echo $(YELLOW)"  make run or run_cpp"$(NOCOLOR)"- runs the application";
+	@echo $(YELLOW)"  make help "$(NOCOLOR)"- shows this message";
+	@echo $(YELLOW)"  make all "$(NOCOLOR)"- makes everything needed to run but doesn't run";
+	@echo $(YELLOW)"  make data "$(NOCOLOR)"- downloads data as needed";
+	@echo $(YELLOW)"  make deps "$(NOCOLOR)"- makes/prepares dependencies";
+	@echo $(YELLOW)"  make install-reqs "$(NOCOLOR)"- Installs requirements needed to run this sample on your system.";
+	@echo $(YELLOW)"  make uninstall-reqs "$(NOCOLOR)"- Uninstalls requirements that were installed by the sample program.";
+	@echo $(YELLOW)"  make default_model "$(NOCOLOR)"- compiles a default model to use when running";
```

Comment:
This target doesn't exist, just displays a make error. Recommended to either remove the target from the help message or add a target that does the appropriate work.